### PR TITLE
Refined QComponent pull request requirements

### PR DESCRIPTION
**Pull request by marcolincs**
### What are the issues this pull addresses (issue numbers / links)?
This change implements the guide changes we discussed the past week.

### CODE REVIEW

**Comment by zlatko-minev** [comment]
Thank you, this is very helpful

**Review by zlatko-minev — APPROVED** [review]

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
Good to add more directions for clarification!!!
Thanks!

File: contributor_guidelines/pull_request_rules.md [hunk 1]
```diff
@@ -4,39 +4,32 @@
...
+    - The method description must also include details of the method signature and returns
+    - The class description should include a details of the `default_options` keys you defined. it should also include an image
+4. Make the doc_strings visible to the online documentation by adding the component to the TOC (table of content)
+    1. Open the file `qiskit_metal/qlibrary/__init__.py`
+    1. Add the QComponent class name to the most appropriate toc-tree section
+    1. Then locate the code line `if config.is_building_docs():` and inside(under) it add an import statement to your new QComponent class.
+5. Add a sample tutorial notebook
+    - Even if you do not want this to appear in the official documentation, the tutorial notebook will be helpful for the general users. Please add this in the tutorials appendix C folder
+    - Make sure the notebook looks the same as the other notebooks in that folder, especially the cmponent rendering lines. Only deviate from a similar component template if your component has some especially unique behavior that is worth highlighting
```
Comment (priti-ashvin-shah-ibm) [inline contributor_guidelines/pull_request_rules.md:28]: cmponent vs component

**Review by zlatko-minev — APPROVED** [review]

**Comment by zlatko-minev** [comment]
What else do we need to merge?

**Pull request merged**